dashboard/common/utils.py

```diff
@@ -86,9 +86,9 @@ def get_myla_globals(request):
         username = current_user.get_username()
         user_courses_info = get_user_courses_info(username, course_id)
 
-    if settings.LOGIN_URL:
+    if not settings.STUDENT_DASHBOARD_LTI:
         login_url = settings.LOGIN_URL
-    if settings.LOGOUT_URL:
+    if not settings.STUDENT_DASHBOARD_LTI:
         logout_url = settings.LOGOUT_URL
```

Comment:
For some reason, when I try to access `localhost:5001`, it's doing that looping thing which I've pointed out before: it keeps trying and failing to load. This isn't happening on `master` (you just get a 404 when redirected to `accounts/login`). I can't explain this yet. `myla_globals['login_url']` must be in use somewhere and the logic you changed is breaking something?

Reply:
Just a note to say that this is successfully hiding the "Logout" button, but the behavior around visiting the domain and using the default backend seems to have been disrupted somehow.